client/py/aou_workbench_client/auth.py

```diff
@@ -0,0 +1,82 @@
+"""OAuth against the AoU Workbench API using bearer tokens.
+
+The credentials are obtained by oauth2client, from one of:
+*   Google application default credentials (expected case, from notebook servers).
+*   A private key file, path specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+
+For local development using a private key file:
+  api/project.rb get-service-creds --project all-of-us-workbench-test --account $USER@pmi-ops.org
+  GOOGLE_APPLICATION_CREDENTIALS=`pwd`/api/sa-key.json python client/py/aou_workbench_client/auth.py
```

Comment:
Can you put a newline before "python"? (I assume you can set GOOGLE_APPLICATION_CREDENTIALS in a separate command?)

Reply:
Sure. I tend to keep it on one line when I can, to localize state / so I don't need to double-check that I've defined the right variable in the right shell.